### DISCUSSION TITLE
None of the pipes seem to be working

### DISCUSSION THREAD

**Discussion by BIZKIT551** [KRTirtho/spotube#636] [Q&A]
I've tried all the pipes with both YouTube and YouTube Music API but none are working for me.
Some songs load forever and will never download and some will play but will show a red triangle when trying to download. 

The same happens without using any pipes and using just the YouTube API.

Not sure if this is only happening to me, but I hope that there can be a solution.![Screenshot_20230819_043222.jpg](https://github.com/KRTirtho/spotube/assets/18755473/31c9dcf6-054d-4ed8-8665-1e899226305f)

**Comment by KRTirtho** [comment]
Try changing the Piped Instance to **drgns.space**. YouTube API based downloads usually a reason for IP block so the downloader always defaults to Piped API. But unfortunately Piped is broken now :'(

**Comment by AbduDevUz** [comment]
Am I allowed to use this too?
